src/components/TempTodo.tsx

```diff
@@ -0,0 +1,42 @@
+import { Todo } from '../types';
+
+type Props = {
+  tempTodo: Todo,
+  handleDeleteTodo: (value: number) => void,
+};
+
+export const TempTodo: React.FC<Props> = ({ tempTodo, handleDeleteTodo }) => {
+  return (
+    <li
+      className="todo"
+      key={tempTodo.id}
```

Comment:
The key attribute is not needed here as the component is not being iterated. It is usually used within a map() function to uniquely identify each component. 

Also return can be deleted, just leave `( )`